main.go

```diff
@@ -275,6 +279,12 @@ func metricsServer(registry prometheus.Gatherer, port int) {
 		w.WriteHeader(200)
 		w.Write([]byte("ok"))
 	})
+	// Add buildInfo
+	mux.HandleFunc("/build", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
This is generally an antipattern, we don't want to reveal the version that is running to any user. I think it's good enough to log the version on start up.

Reply:
Agreed with @brancz . We do not need this http endpoint for querying version info.